src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Controllers/AuthController.cs

```diff
@@ -124,7 +124,7 @@ private async Task SendConfirmationEmail(SendConfirmationEmailRequestDto sendCon
 #if BlazorServer
         resetPasswordLink = $"{_appSettings.Value.WebServerAddress}{resetPasswordLink}";
 #else
-        resetPasswordLink = $"{_server.GetHostUri()}{resetPasswordLink}";
+        resetPasswordLink = $"{new Uri($"{HttpContext.Request.Scheme}://{HttpContext.Request.Host}{HttpContext.Request.PathBase}")}{resetPasswordLink}";
```

Comment:
this change is not related to this issue.
plz remove it and submit a new issue.